.github/actions/setup-database.yml

```diff
@@ -0,0 +1,89 @@
+# evennia/setup-database
+
+# Use this action in a workflow for when you need to do initialization of database services
+# (such as with PostgreSQL and MySQL) before you initiate unit tests that will employ that
+# database service.
+
+# NOTE: This action was intended for use with the core Evennia workflows ONLY.
+
+name: Set up Evennia database service
+author: dvoraen
+description: "Activates the database server for the passed in service and ensures it's ready for use."
+
+inputs:
+  database:
+    description: "Database service being initialized."
+    type: string
+    required: true
+
+runs:
+  using: "composite"
+
+  steps:
+    # - name: Set up SQLite
+    #   if: inputs.database == "sqlite3"
```

Comment:
Left over debugs?

Reply:
I left that there as a reminder in case SQLite requires any additional configuration in the future.  I can remove it if you'd rather it not exist as a ghost.